From 8246d7e5a80c3df4ca97d413ae40308d6970b38c Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Wed, 24 Sep 2014 13:31:19 +0800
Subject: [PATCH 0113/5242] MLK-9606-1 usb: otg: test: add otg test device
 into otg and EH TPL

commit  224768be9eb2988669b849bfcc47b9242f8da086 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch add otg test device with VID(0x1a0a) and PID(0x0200) into otg and
EH TPL.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit b88ee5e2cbf55aaed3b6c5dd0aff7f826f9e357d)
(cherry picked from commit 8c11515f82876e69aa14e1c31fcb5031cf4edfcc)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/core/otg_whitelist.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/core/otg_whitelist.h b/drivers/usb/core/otg_whitelist.h
index 9d08961..039aa13 100644
--- a/drivers/usb/core/otg_whitelist.h
+++ b/drivers/usb/core/otg_whitelist.h
@@ -23,6 +23,7 @@
 #define TEST_HS_HOST_PORT_SUSPEND_RESUME	0x0106
 #define TEST_SINGLE_STEP_GET_DEV_DESC		0x0107
 #define TEST_SINGLE_STEP_SET_FEATURE		0x0108
+#define TEST_OTG_TEST_DEVICE_SUPPORT		0x0200
 { USB_DEVICE(0x1a0a, TEST_SE0_NAK_PID) },
 { USB_DEVICE(0x1a0a, TEST_J_PID) },
 { USB_DEVICE(0x1a0a, TEST_K_PID) },
@@ -30,6 +31,7 @@
 { USB_DEVICE(0x1a0a, TEST_HS_HOST_PORT_SUSPEND_RESUME) },
 { USB_DEVICE(0x1a0a, TEST_SINGLE_STEP_GET_DEV_DESC) },
 { USB_DEVICE(0x1a0a, TEST_SINGLE_STEP_SET_FEATURE) },
+{ USB_DEVICE(0x1a0a, TEST_OTG_TEST_DEVICE_SUPPORT) },
 #endif
 
 #define USB_INTERFACE_CLASS_INFO(cl) \
-- 
1.7.9.5

